piccolo_admin/endpoints.py

```diff
@@ -388,6 +392,20 @@ def __init__(
             ),
         )
 
+        auth_app.mount(
+            path="/change-password/",
+            app=AuthenticationMiddleware(
+                change_password(login_url=f"{self.prefix_path}auth/login/"),
+                SessionsAuthBackend(),
+            ),
+        )
```

Comment:
I would consider mounting this on the `api_app` as it already has authentication enabled.